### PR TITLE
Update raphael.json

### PR DESCRIPTION
**Pull request by raphael-as**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ DONE] The file is in the `domains` folder and is in the JSON format.
- [DONE ] You have completed your website. 
- [ DONE] The website is reachable.  
- [DONE ] You're not using Vercel or Netlify.  
- [DONE ] The CNAME record doesn't contain `https://` or `/`.  
- [ DONE] There is sufficient information at the `owner` field.  

## Website Link/Preview
[Website](https://raphael-as.github.io)

### CODE REVIEW

File: domains/raphael.json [hunk 1]
```diff
@@ -9,6 +9,9 @@
             "185.199.109.153",
             "185.199.110.153",
             "185.199.111.153"
+        ],
+        "CNAME": [
+            "raphael-as.github.io."
         ]
```
Comment (CuteDog5695) [inline domains/raphael.json:15]: you dont need to put a array for CNAME, so remove the []

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/raphael.json [hunk 2]
```diff
@@ -10,8 +10,6 @@
             "185.199.110.153",
             "185.199.111.153"
         ],
-        "CNAME": [
-            "raphael-as.github.io."
-        ]
+        "CNAME": "raphael-as.github.io.",
```
[the remark this answers is not in the document]
Comment (CuteDog5695) [reply domains/raphael.json:13]: oh and you cant use CNAME with any other records, so remove A record

File: domains/raphael.json [hunk 3]
```diff
@@ -4,11 +4,6 @@
         "email": "raphaelas.pro@gmail.com"
     },
     "record": {
-        "A": [
-            "185.199.108.153",
-            "185.199.109.153",
-            "185.199.110.153",
-            "185.199.111.153"
-        ]
+        "CNAME": "raphael-as.github.io."
```
Comment (CuteDog5695) [inline domains/raphael.json:7]: remove the . from raphael-as.github.io

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/raphael.json (hunk 3 above)
Comment (raphael-as) [reply domains/raphael.json:7]: sorry haha, i just copy past from the github suggested cname

**Review by iostpa — APPROVED** [review]
Now it seems good to me

**Pull request merged**